src/lib/helpers/jsToAqua.ts

```diff
@@ -152,13 +200,15 @@ export const jsToAquaImpl = (
       return error("All array elements must be of the same type");
     }
 
+    const ind = INDENTATION.repeat(newNestingLevel);
```

Comment:
I suggest to declare two similar variables:
```suggestion
    const newIndent = INDENTATION.repeat(newNestingLevel);
    const prevIndent = INDENTATION.repeat(nestingLevel);
```

And use them in the `value`. Right now, it is a variable in one place, in-place expression in another.